src/analyze/mod.rs

```diff
@@ -60,28 +64,53 @@ pub fn render_explain(explain: &Analysis) -> anyhow::Result<()>
 pub async fn command(cli: &mut Connection, options: &Analyze)
     -> anyhow::Result<()>
 {
-    let Some(inner_query) = &options.query else {
-        anyhow::bail!("Query argument is required");
-    };
-    let query = if classify::is_analyze(inner_query) {
-        // allow specifying options in the query itself
-        Cow::Borrowed(inner_query)
+    let data = if let Some(json_path) = &options.read_json {
+        fs::read_to_string(&json_path).await
+            .with_context(|| format!("cannot read {json_path:?}"))?
+
     } else {
-        // but also do not make user writing `analyze` twice
-        Cow::Owned(format!("analyze {inner_query}"))
-    };
+        let Some(inner_query) = &options.query else {
+            anyhow::bail!("Query argument is required");
+        };
+        let query = if classify::is_analyze(inner_query) {
+            // allow specifying options in the query itself
+            Cow::Borrowed(inner_query)
+        } else {
+            // but also do not make user writing `analyze` twice
+            Cow::Owned(format!("analyze {inner_query}"))
+        };
 
-    let data = cli.query_required_single::<String, _>(&query, &()).await?;
+        cli.query_required_single::<String, _>(&query, &()).await?
+    };
+    if let Some(out_path) = &options.debug_output_file {
+        if out_path == Path::new("-") {
+            let mut out = io::stdout();
+            out.write_all(data.as_bytes()).await?;
+            out.flush().await?;
+        } else if out_path.starts_with("/dev") {
+            async {
+                let mut out = fs::File::create(&out_path).await?;
```

Comment:
This is for `/dev/stdout`?

Reply:
Did you meant to write `} else if !out_path.starts_with("/dev") {` with the `not`?